### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by seitosan**

### CODE REVIEW

File: .artifakt.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+title: Wordpress on Ovh
+description:  Wordpress on Ovh  with terraform
```
Comment (ninuxio) [inline .artifakt.yml]: double spaces

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+*terraform
```
Comment (ninuxio) [inline .gitignore]: prefered .terraform

File: README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#  Wordpress  on OVH
+
+This artifact deploys a demo Wordpress on  OVH using openstack infrastructure.
```
Comment (ninuxio) [inline README.md]: deploy

Comment (ninuxio) [reply README.md]: pas demo

Comment (ninuxio) [reply README.md]: double espace

File: README.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+We have created a security group so that the instance only responds on port 80(HTTP).
```
Comment (ninuxio) [inline README.md]: the instance only responds on port 80(HTTP).

File: outputs.tf [hunk 5]
```diff
@@ -0,0 +1,3 @@
+output "publicUrl" {
```
Comment (ninuxio) [inline outputs.tf]: miss an _

**Comment by seitosan** [comment]
Done

**Pull request merged**